content/v2/csidriver/features/powerscale.md

```diff
@@ -1,570 +1,659 @@
+```
+
+## Volume Snapshot Feature
+
+The CSI PowerScale driver version 2.0 and later supports managing v1 snapshots. 
+
+In order to use Volume Snapshots, ensure the following components have been deployed to your cluster:
+
+- Kubernetes Volume Snapshot CRDs
+- Volume Snapshot Controller
+
+> For general use, update the snapshot controller YAMLs with an appropriate namespace before installing. For
+> example, on a Vanilla Kubernetes cluster, update the namespace from default to kube-system before issuing the
+> kubectl create command.
+
+### Volume Snapshot Class
+
+During the installation of CSI PowerScale driver version 2.0 and higher, no default Volume Snapshot Class will get created.
+
+Following are the manifests for the Volume Snapshot Class:
+
+1. VolumeSnapshotClass 
+```yaml
+
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: "isilon-snapclass"
+driver: csi-isilon.dellemc.com
+#The deletionPolicy of a volume snapshot class can either be Retain or Delete
+#If the deletionPolicy is Delete, then the underlying storage snapshot is deleted along with the VolumeSnapshotContent object.
+#If the deletionPolicy is Retain, then both the underlying snapshot and VolumeSnapshotContent remain
+deletionPolicy: Delete
+parameters:
+  #IsiPath should match with respective storageClass IsiPath
+  IsiPath: "/ifs/data/csi"
+```
+
+The following is a sample manifest for creating a Volume Snapshot using the **v1** snapshot APIs; The following snippet assumes that the persistent volume claim name is testvolume.
+
+```yaml  
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: pvcsnap
+  namespace: default
+spec:
+  volumeSnapshotClassName: isilon-snapclass
+  source:
+    persistentVolumeClaimName: testvolume
+```
+
+Once the VolumeSnapshot has been successfully created by the CSI PowerScale driver, a VolumeSnapshotContent object is automatically created. Once the status of the VolumeSnapshot object has the _readyToUse_ field set to _true_ , it is available for use.
+
+Following is the relevant section of VolumeSnapshot object status:
+
+```yaml
+status:
+  boundVolumeSnapshotContentName: snapcontent-xxxxxxxxxxxxx
+  creationTime: "2020-07-16T08:42:12Z"
+  readyToUse: true
+```
+
+### Creating PVCs with Volume Snapshots as Source
+
+The following is a sample manifest for creating a PVC with a VolumeSnapshot as a source:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: createfromsnap
+  namespace: default
+spec:
+  storageClassName: isilon
+  dataSource:
+    name: pvcsnap
+    kind: VolumeSnapshot
+    apiGroup: snapshot.storage.k8s.io
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+> Starting from CSI PowerScale driver version 2.2, it is allowed to create PersistentVolumeClaim from VolumeSnapshot with different isi paths i.e., isi paths of the new volume and the VolumeSnapshot can be different.

```

Comment:
Change sentence to read "...different isi paths can be used to create PersistentVolumeClaim from VolumeSnapshot. This means the isi paths of the new volume and the VolumeSnapshot can be different.

Reply:
Addressed the comment